cmd/buildah/addcopy.go

```diff
@@ -10,25 +10,26 @@ var (
 	addFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "name",
-			Usage: "name or ID of the working container",
+			Usage: "name or `ID` of the working container",
```

Comment:
Ok, I have no idea, I new it was not markdown, I thought you were just adding emphasis.